crates/astria-composer/src/searcher/executor.rs

```diff
@@ -252,6 +360,30 @@ impl Executor {
         Ok(())
     }
 }
+
+#[derive(Debug, thiserror::Error)]
+enum ExecutionError {
+    #[error("failed to submit sequencer transaction")]
+    TransactionSubmissionFailed {
+        error: SequencerClientError,
+        transaction: SignedTransaction,
```

Comment:
You don't seem to be doing anything with these transactions but only use them for reporting. If that's the case start a tracing span with a field identifying the tx instead of moving the object around

Reply:
yeah maybe use the hash (same for errs below)

Reply:
logging the hash of the signed tx now